src/pages/policy/input/ParameterOverTime.jsx

```diff
@@ -1,18 +1,23 @@
 import Plot from "react-plotly.js";
 import { ChartLogo } from "../../../api/charts";
-import {
-  getSortedParameterValues,
-  getReformedParameter,
-} from "../../../api/parameters";
 import { getPlotlyAxisFormat } from "../../../api/variables";
 import useMobile from "../../../layout/Responsive";
 import useWindowHeight from "layout/WindowHeight";
 import style from "../../../style";
 import { plotLayoutFont } from "pages/policy/output/utils";
-import { localeCode } from "lang/format";
+import { localeCode } from "lang/format"; /**
+ *
+ * @param {object} policy the policy object
+ * @returns the reform policy label
+ */
+function getReformPolicyLabel(policy) {
+  const urlParams = new URLSearchParams(window.location.search);
+  const reformPolicyId = urlParams.get("reform");
+  return reformPolicyId ? `Policy #${reformPolicyId}` : "reform";
```

Comment:
Instead of this, could you display the full name here if there is one, followed by the policy number if there isn't a name? And perhaps there's a good way to truncate this if the name is too long?